src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -521,12 +521,13 @@ To prevent dividing by zero, we change AI/(AI x dz)" to
 """
 function make_compute_exp_tendency(model::PlantHydraulicsModel, canopy)
     function compute_exp_tendency!(dY, Y, p, t)
+        FT = eltype(p.canopy.hydraulics.area_index.root)
```

Comment:
in cases like this, should we get FT from the model (or another struct) instead? it seems a little cleaner to me.

Reply:
sure! I'll change this to get FT from the model and try to find other instances like this